packages/tools/examples/StackSegmentation/index.ts

```diff
@@ -0,0 +1,385 @@
+);
+
+const size = '500px';
+const content = document.getElementById('content');
+const viewportGrid = document.createElement('div');
+
+viewportGrid.style.display = 'flex';
+viewportGrid.style.display = 'flex';
+viewportGrid.style.flexDirection = 'row';
+
+const element1 = document.createElement('div');
+element1.oncontextmenu = () => false;
+
+element1.style.width = size;
+element1.style.height = size;
+
+viewportGrid.appendChild(element1);
+
+const element2 = document.createElement('div');
+element2.oncontextmenu = () => false;
+
+element2.style.width = size;
+element2.style.height = size;
+
+viewportGrid.appendChild(element2);
+
+content.appendChild(viewportGrid);
+
+const instructions = document.createElement('p');
+instructions.innerText = 'Click the image to rotate it.';
+
+content.append(instructions);
+
+async function getImageIds() {
+  // Get Cornerstone imageIds and fetch metadata into RAM
+  const imageIds = await createImageIdsAndCacheMetaData({
+    StudyInstanceUID:
+      '1.3.6.1.4.1.14519.5.2.1.7009.2403.871108593056125491804754960339',
+    SeriesInstanceUID:
+      '1.3.6.1.4.1.14519.5.2.1.7009.2403.367700692008930469189923116409',
+    wadoRsRoot: 'https://domvja9iplmyu.cloudfront.net/dicomweb',
+  });
+
+  return imageIds;
+}
+
+function setupTools(toolGroupId) {
+  // Add tools to Cornerstone3D
+  cornerstoneTools.addTool(WindowLevelTool);
+  cornerstoneTools.addTool(StackScrollMouseWheelTool);
+  cornerstoneTools.addTool(ZoomTool);
+  cornerstoneTools.addTool(SegmentationDisplayTool);
+
+  // Define a tool group, which defines how mouse events map to tool commands for
+  // Any viewport using the group
+  const toolGroup = ToolGroupManager.createToolGroup(toolGroupId);
+
+  // Add the tools to the tool group and specify which volume they are pointing at
+  toolGroup.addTool(StackScrollMouseWheelTool.toolName, { loop: true });
+  toolGroup.addTool(ZoomTool.toolName);
+  toolGroup.addTool(WindowLevelTool.toolName);
+  toolGroup.addTool(SegmentationDisplayTool.toolName);
+
+  toolGroup.setToolEnabled(SegmentationDisplayTool.toolName);
+
+  toolGroup.setToolActive(ZoomTool.toolName, {
+    bindings: [
+      {
+        mouseButton: MouseBindings.Secondary, // Right Click
+      },
+    ],
+  });
+  toolGroup.setToolActive(WindowLevelTool.toolName, {
+    bindings: [
+      {
+        mouseButton: MouseBindings.Primary, // Middle Click
+      },
+    ],
+  });
+  toolGroup.setToolActive(StackScrollMouseWheelTool.toolName);
+  return toolGroup;
+}
+// ============================= //
+
+/**
+ * Adds two concentric circles to each axial slice of the demo segmentation.
+ */
+function createMockEllipsoidSegmentation(dimensions, imageIds) {
+  const center = [dimensions[0] / 2, dimensions[1] / 2, dimensions[2] / 2];
+  const outerRadius = 64;
+  const innerRadius = 32;
+
+  for (let z = 0; z < dimensions[2]; z++) {
+    let voxelIndex = 0;
+    const { image } = cache.getCachedImageBasedOnImageURI(imageIds[z]);
```

Comment:
You know how we have a `cache.getVolume`, we should have a `cache.getImage` too instead of this `getCachedImage.....`